torch/nn/quantized/modules/conv.py

```diff
@@ -116,6 +116,25 @@ def _save_to_state_dict(self, destination, prefix, keep_vars):
         if self.bias is not None:
             destination[prefix + 'bias'] = self.bias
 
+    @torch.jit.export
+    def __getstate__(self):
+        return (
+            self.in_channels,
+            self.out_channels,
+            self.kernel_size,
+            self.stride,
+            self.padding,
+            self.dilation,
+            self.transposed,
+            self.output_padding,
+            self.groups,
+            self.padding_mode,
+            self.weight(),
+            self.bias,
```

Comment:
This can be none. Is special handling requires to handle that?

Reply:
No, I don't know why we had special handling before, `None`s are directly representable in both the Python and TorchScript pickle formats